modules/core/src/main/java/org/apache/ignite/internal/visor/metric/VisorMetricTask.java

```diff
@@ -69,6 +70,8 @@ protected VisorMetricJob(@Nullable VisorMetricTaskArg arg, boolean debug) {
             if (arg == null)
                 throw new IgniteException("VisorMetricTaskArg is null");
 
+            ignite.context().security().authorize(ADMIN_VIEW);
```

Comment:
Do we really need this check here? In my understanding, visor task should be already authorized before running.

Reply:
honestly I have no idea where it should be, I made it by analogy with other tasks